src/App.tsx

```diff
@@ -1,7 +1,41 @@
-import React from 'react';
+import { Component, ReactNode } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey : string;
+};
+
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    pressedKey: '',
+  };
+
+  initialText = 'Nothing was pressed yet';
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handleKeyPressed);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.handleKeyPressed);
+  }
+
+  handleKeyPressed = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: `${event.key}` });
+  };
+
+  render(): ReactNode {
+    const { pressedKey } = this.state;
+
+    return (
+      <div className="App">
+        <p className="App__message">
+          {(pressedKey.length > 0) ? (
+            `The last pressed key is [${pressedKey}]`
+          ) : (
+            this.initialText
+          )}
```

Comment:
```suggestion
          {(pressedKey.length > 0) 
            ? `The last pressed key is [${pressedKey}]`
            : this.initialText
          )}
```
![image](https://user-images.githubusercontent.com/96409563/207406540-48deface-bee2-41f9-b0d8-0440ecc2a207.png)